packages/shared/src/utils.ts

```
@@ -0,0 +1,329 @@
+function getPriorityColor(priority?: number, provider: string = 'discord'): string | number {
+  const p = priority || 3;
+  
+  switch (provider) {
+    case 'discord':
+      const discordColors = {
+        1: 0x95a5a6, // gray
+        2: 0x3498db, // blue
+        3: 0x2ecc71, // green
+        4: 0xf39c12, // orange
+        5: 0xe74c3c, // red
+      };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)